assets/js/blocks/active-filters/edit.js

```diff
@@ -72,20 +73,14 @@ const Edit = ( { attributes, setAttributes } ) => {
 		);
 	};
 
-	const TagName = `h${ headingLevel }`;
-
 	return (
 		<div className={ className }>
 			{ getInspectorControls() }
-			<TagName>
-				<PlainText
-					className="wc-block-attribute-filter-heading"
```

Comment:
Should we keep the `className` prop and change it to `wc-block-active-filters-heading`?

Reply:
Actually, thinking about it twice I would still pass the class name from here, but I would set it to `wc-block-filters-heading`, so you can reuse the same class name for all three blocks (so you don't need to add three classes in #1241).

Passing down the class name from here instead of hardcoding it in the `<BlockTitle>` component would allow us to use that component for other blocks which are not filters. Does that sound like a good solution?

Reply:
Passing `wc-block-filter-heading` would limit it to only filters. What I'm hoping to do is to isolate classnames, having `wc-block-filters-heading` inside `blocks/price-filter/edit.js` wouldn’t cut it because that any class inside `price-filter.js` should be scoped to that file and used inside `blocks/price-filter/style.scss`, this is so we don’t create dependencies between blocks and components.